src/js/dp/search.js

```diff
@@ -304,4 +304,34 @@ if (searchContainer) {
       });
     });
   };
-}
+
+  //capture focus in checkboxes
+  const showResultsBtn = document.getElementById('show-results');
+  const checkboxes = document.querySelectorAll('input[type="checkbox"]:not([disabled])');
+  const firstCheckbox = checkboxes[0];
+  const keycodeTab = 9;
+
+  if (showResultsBtn) {
+    document.addEventListener('keydown', function (e) {
+      if (e.key === 'Tab' || e.code === keycodeTab.toString()) {
+        if (document.activeElement === showResultsBtn) {
+          firstCheckbox.focus();
+          e.preventDefault();
+        }
+      }
+    });
+
+    //tab to checkboxes after filtering results
+    const keycodeEnter = 13
+    const filterBtn = document.getElementById("filter-results");
+    if (filterBtn) {
+      document.addEventListener('keyup', function (e) {
+        if (e.key === 'Enter' || e.code === keycodeEnter.toString()) {
+          if (document.activeElement === filterBtn) {
+            firstCheckbox.focus();
+          }
```

Comment:
I think this should be triggered on spacebar as well, as it's a button: https://webaim.org/techniques/keyboard/#:~:text=A%20keyboard%20user%20typically%20uses,or%20manipulated%20with%20the%20keyboard.